types/open-api/identity_params.yaml

```diff
@@ -15,3 +15,9 @@ components:
             - type: string
               format: uuid
             - type: "null"
+        forceAnonymous:
+          deprecated: true # confirm this, I just noticed it was deprecated in the 'thread' docs so mirrored that here
```

Comment:
And one more question! `forceAnonymous`: deprecated? (if so I'll remove it from the tests, I ended up modeling the comment shape on what I was seeing go through on the live site when commenting, but if this is going away I'll strip it out)

Reply:
It is indeed deprecated. In the original BobaBoard design, we were supposed to have identity reveal through "friendship", and this was supposed to be a signal that the post had to be completely anonymous, friendship notwithstanding. We ended up not implementing that mechanism and (while it's still planned to have that possibility), it will likely look very different from a simple "force anonymous" boolean. 

I don't think you should remove the tests you have already written until we actually remove it from the codebase though.